docs/guides/classical-feedforward-and-control-flow.ipynb

```diff
@@ -312,10 +312,10 @@
     "- Limitations on classical registers:\n",
     "      - The operand used in an `if_test` statement must be 32 or fewer bits. Thus, if you are comparing an entire `ClassicalRegister`, the size of that `ClassicalRegister` must be 32 or fewer bits. If you are comparing just a single bit from a `ClassicalRegister`, however, that `ClassicalRegister` can be of any size (since the operand is only one bit).\n",
```

Comment:
Since this is now the only sub-bullet, it no longer needs to be a sub-bullet 😄 - i.e. this should just be 

> - Due to the limited physical memory  ...
> - The operand used in an if_test statement must be 32  ...
> - Nested conditionals are not allowed ...